### DISCUSSION TITLE
Error: Maximum call stack size exceeded.

### DISCUSSION THREAD

**Discussion by kennedycraig** [AntlerVC/firetable#753] [Support / Q&A] [answered]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Rowy Run version**
If applicable, share the Rowy Run version from your project settings.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by BeeBombshell** [comment, marked as answer]
Hi @kennedycraig, I'm not sure I understand. We might be able to give you a better guide if you could provide some form of explanation of the problem you're having.
